tools/ports/harfbuzz.py

```diff
@@ -0,0 +1,38 @@
+import os, shutil, logging, subprocess
+
+TAG = '1.7.5'
+
+def get(ports, settings, shared):
+  if settings.USE_HARFBUZZ == 1:
+    ports.fetch_project('harfbuzz', 'https://github.com/harfbuzz/harfbuzz/archive/' + TAG + '.zip', 'harfbuzz-' + TAG)
```

Comment:
Any change we use .tar.bz2 files here so using official harfbuzz release tarballs?

Reply:
Right now the code in `tools/system_libs.py` uses ZipFile, so it's specific to zip. But it would be nice to extend that.

Reply:
Will try to use https://stackoverflow.com/a/29416112